### PR TITLE
USWDS-Site: Create changelog for USWDS#5423

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Creates changelog for https://github.com/uswds/uswds/pull/5423

## Related issue

https://github.com/uswds/uswds/issues/5401

## Preview link

[Card changelog →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-card-header-first-media-changelog/components/card/#latest-updates)

### CODE REVIEW

File: _data/changelogs/component-card.yml [hunk 1]
```diff
@@ -2,6 +2,12 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a margin bug that causes card media to overlap headers on header-first variants.
```
Comment (amyleadem) [inline _data/changelogs/component-card.yml:6]: Just a small re-ordering tweak for clarity. 
```suggestion
    summary: Fixed a margin bug on header-first variants that causes card media to overlap headers.
```

Comment (mahoneycm) [reply _data/changelogs/component-card.yml]: Good point! Commited and resolved in [fbc1876](https://github.com/uswds/uswds-site/pull/2227/commits/fbc1876acd3731b2ead6c7daa316789f0d62cee6)

Comment (mahoneycm) [reply _data/changelogs/component-card.yml]: Good point! Commited and resolved in [fbc1876](https://github.com/uswds/uswds-site/pull/2227/commits/fbc1876acd3731b2ead6c7daa316789f0d62cee6)

File: _data/changelogs/component-card.yml [hunk 2]
```diff
@@ -2,6 +2,12 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a margin bug on header-first variants that causes card media to overlap headers.
```
Comment (amyleadem) [inline _data/changelogs/component-card.yml:6]:
```suggestion
    summary: Fixed a margin bug on header-first variants that caused exdent card media to overlap headers.
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
Added just a couple more small tweaks. Other than that, looks good!

**Review by amyleadem — APPROVED** [review]

**Review by mejiaj — APPROVED** [review]

**Pull request merged**